products/views.py

```diff
@@ -1,3 +1,42 @@
-from django.shortcuts import render
+from django.http.response  import JsonResponse
+from django.views          import View
+from django.http.response  import JsonResponse
+from products.models       import Product 
+
+
+class SubCategoryView(View):
+    def get(self, request, sub_category_id):
+        products = Product.objects.filter(sub_category_id = sub_category_id)
+        results = [{
+                        'product_id'         : product.id,
+                        'kr_name'            : product.kr_name,
+                        'en_name'            : product.en_name,
+                        'price'              : product.price,
+                        'sub_category_id'    : product.sub_category.id,
+                        'sub_category_name'  : product.sub_category.kr_name,
+                        'main_category_name' : product.sub_category.main_category.name,
+                        'main_category_id'   : product.sub_category.main_category.id,
+                        'images' : [{
+                            "image_id"       : image.id,
+                            "product_image"  : image.url,
+                            } for image in product.image_set.all()],
+                        'rating'        : product.rating
+                    }for product in products]
+
+        return JsonResponse({"result":results}, status=200)
+
+class ProductDetailView(View):
+    def get(self, request, product_id):
+        products = Product.objects.filter(id = product_id)
+        results =[{
+            'product_id'        : product.id,
+            'kr_name'           : product.kr_name,
+            'en_name'           : product.en_name,
+            'price'             : product.price,
+            'sub_category_id'   : product.sub_category.id,
+            'sub_category_name' : product.sub_category.kr_name,
+            'rating'            : product.rating,
+            'description_img'   : product.description
+        } for product in products]
+        return JsonResponse({"result":results}, status=200) 
```

Comment:
하나의 기능만 담은 브랜치로 푸시해주세요
```suggestion
class ProductDetailView(View):
    def get(self, request, product_id):
        products = Product.objects.filter(id = product_id)
        results =[{
            'product_id'        : product.id,
            'kr_name'           : product.kr_name,
            'en_name'           : product.en_name,
            'price'             : product.price,
            'sub_category_id'   : product.sub_category.id,
            'sub_category_name' : product.sub_category.kr_name,
            'rating'            : product.rating,
            'description_img'   : product.description
        } for product in products]
        return JsonResponse({"result":results}, status=200) 
```